lib/lighthouse/veteran_verification/configuration.rb

```diff
@@ -120,7 +119,7 @@ def token_service(lighthouse_client_id, lighthouse_rsa_key_path, aud_claim_url =
       aud_claim_url ||= settings.aud_claim_url
 
       @token_service ||= Auth::ClientCredentials::Service.new(
-        url, API_SCOPES, lighthouse_client_id, aud_claim_url, lighthouse_rsa_key_path
+        url, API_SCOPES, lighthouse_client_id, aud_claim_url, lighthouse_rsa_key_path, 'veteran-verification'
```

Comment:
Note: This will opt-in to using the redis store to cache the access token for this service. The reason for the changes contained in this PR is because Lighthouse wants us to cache the access token and only request a new one when the old one expires